pkg/virt-controller/watch/vm.go

```diff
@@ -2457,6 +2453,9 @@ func (c *VMController) updateStatus(vmOrig *virtv1.VirtualMachine, vmi *virtv1.V
 	}
 	vm.Status.Ready = ready
 
+	runStrategy, _ := vmOrig.RunStrategy()
```

Comment:
maybe checking the error?
```
if err != nil {	
	return vm, &syncErrorImpl{fmt.Errorf(fetchingRunStrategyErrFmt, err), FailedCreateReason}	
}
```

Reply:
That's deliberate, I don't really care if I get an error here, the worst that can happen is that I store 'RunStrategyUnknown' which is the correct state for this field anyway.